apps/web-giddh/src/app/gst/filing/filing.component.html

```diff
@@ -1,8 +1,8 @@
 <ng-container appTranslate [file]="'gst'" (localeData)="localeData = $event"
     (commonLocaleData)="commonLocaleData = $event" (translationComplete)="translationComplete($event)">
-    <div class="help-support-aside setting-sidebar-wrapper" *ngIf="asideGstSidebarMenuState === 'in'">
-        <gstr-sidebar (navigateEvent)="handleNavigation($event)" [activeCompanyGstNumber]="activeCompanyGstNumber"
-            [selectedGstModule]="selectedGst" [localeData]="localeData" [commonLocaleData]="commonLocaleData" class="gst-sidebar"></gstr-sidebar>
+    <div class="help-support-aside setting-sidebar-wrapper gst-sidebar-wrapper" *ngIf="asideGstSidebarMenuState === 'in'">
+        <gstr-sidebar (navigateEvent)="handleNavigation($event)" [activeCompanyGstNumber]="activeCompanyGstNumber" [selectedGstModule]="selectedGst" class="gst-sidebar" (clickOutside)="isMobileScreen ? asideGstSidebarMenuState = 'out' : ''" [exclude] ="'#primary-new-header'" [commonLocaleData]="commonLocaleData"
```

Comment:
Please don't change the order of input elements.

Reply:
reverted